packages/calypso-e2e/src/lib/components/notifications-component.ts

```diff
@@ -64,6 +75,7 @@ export class NotificationsComponent {
 	 * @returns {Promise<void>} No return value.
 	 */
 	async waitForUndoMessageToDisappear(): Promise< void > {
+		this.waitForUndoMessage();
```

Comment:
**comment:blocking**: FYI, this will not behave in the way you expect, since you're missing the `await` keyword.

Reply:
There's another occurrence of this higher up in the file too.

Reply:
Great catch, Edwin! Just updated it.